src/User.php

```diff
@@ -197,6 +202,13 @@ public function isEntityAssign()
     }
 
 
+    public static function isMassiveActionAllowed(int $items_id): bool
+    {
+        $config = Config::getConfigurationValues('core', ['system_user']);
```

Comment:
On huge lists, it will do many DB requests. Is this value available in `$CFG_GLPI`?

Reply:
yes =) i added a commit related to this change